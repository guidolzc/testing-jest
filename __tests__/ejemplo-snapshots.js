const cliente = {
    nombre: 'juan pablo',
    balanca: 500,
    tipo :'Premium'
};

describe('testing al cliente', ()=> {
    test ('es juan pablo', ()=> {
        expect(cliente).toMatchSnapshot(); 
        // esto crea una carpeta como json para actualizar necesitarmos escribiremos npm t-- -u  'para agregar otro cliente o cambiar tendriamos q eliminar la carmeta tambien

        
    });
})

