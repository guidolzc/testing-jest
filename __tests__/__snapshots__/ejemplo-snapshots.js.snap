// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing al cliente es juan pablo 1`] = `
Object {
  "balanca": 500,
  "nombre": "juan pablo",
  "tipo": "Premium",
}
`;
